include/openthread/ip6.h

```diff
@@ -438,6 +438,23 @@ otMessage *otIp6NewMessageFromBuffer(otInstance              *aInstance,
                                      uint16_t                 aDataLength,
                                      const otMessageSettings *aSettings);
 
+/**
+ * Checks whether the given message may be looped back in a case of a multicast destination address.
+ *
+ * @param[in]  aMessage A pointer to the message.
+ *
+ */
```

Comment:
How about adding something like this to clarify when this is set on an `otMessage` and also provided in an accompanying `otMessageInfo`:

```suggestion
 * Indicates whether the given message may be looped back in a case of a multicast destination address.
 *
 * If @p aMessage is used along with an `otMessageInfo`, the `mMulticastLoop` field from `otMessageInfo` structure 
 * takes precedence and will be used instead of the the value set on @p aMessage.
 * 
 * This API is mainly intended for use along with `otIp6Send()` which expects an already prepared IPv6 message.
 *
 * @param[in]  aMessage A pointer to the message.
 *
 */
```